problemsets-mat223/modules/module15.tex

```diff
@@ -127,6 +154,27 @@
 \begin{example}
 	Find the eigenvectors/values of $A=\mat{1&2\\3&4}$.
 
-	XXX Finish
+	First, find the roots of $\Char(A)$ by setting it to $0$.
+	\begin{align*}
+	    \Char(A)&=\det\left(\mat{1-\lambda&2\\3&4-\lambda}\right)\\
+	            &=(1-\lambda)(4-\lambda)-6=\lambda^2-5\lambda-2=0
+	\end{align*}
+	By the quadratic formula\footnote{ Recall that the
+	roots of $ax^2+bx+c$ are given by $\frac{-b\pm\sqrt{b^2-4ac}}{2a}$.}, we find that
+	\[\lambda_1=\frac{5-\sqrt{33}}{2}\qquad\lambda_2=\frac{5+\sqrt{33}}{2}\]
+	are the roots of $\Char(A)$.
+	
+	
+	Following the procedure outlined above, we need to
+	find $\Null(A-\lambda_1I)$ and $\Null(A-\lambda_2I)$.
+	
+	We will start by applying row reduction to $A-\lambda_1I$.
+	\begin{align*}
+	    \mat{1-\frac{5-\sqrt{33}}{2}&2\\3&4-\frac{5-\sqrt{33}}{2}} & \sim
+	    \mat{\frac{-3+\sqrt{33}}{2}&2\\3&\frac{3+\sqrt{33}}{2}} \sim \mat{1&\frac{4}{-3+\sqrt{33}}\\1&\frac{3+\sqrt{33}}{6}}\\
+	    & \sim \matc{1&\frac{4(3+\sqrt{33})}{(-3+\sqrt{33})(3+\sqrt{33})}\\1&\frac{3+\sqrt{33}}{6}} \sim \mat{1&\frac{3+\sqrt{33}}{6}\\1&\frac{3+\sqrt{33}}{6}}\\
+	    & \sim \matc{1&\frac{3+\sqrt{33}}{6}\\0&0}
+	\end{align*}
+	Thus, we conclude that one eigenvector corresponding to eigenvalue $\frac{5-\sqrt{33}}{2}$ is $\vec v_1 = \matc{\frac{3+\sqrt{33}}{6}\\-1}$. Similarly, one eigenvector corresponding to eigenvalue $\frac{5+\sqrt{33}}{2}$ is $\vec v_2=\matc{\frac{3-\sqrt{33}}{6}\\-1}$ is one eigenvector with eigenvalue $\lambda_2 = \frac{5+\sqrt{33}}{2}$.
```

Comment:
Let's change the structure of this sentence to something like:

```
Thus, we conclude that XXX is an eigenvector with eigenvalue YYY. Similarly, ...
```

Reply:
Sure, I also feel that corresponding to is too long ^-^